lib/pollTravis.js

```diff
@@ -172,8 +178,27 @@ function createGhCommentFn (options) {
   }
 }
 
+function createGhStatusFn (options) {
+  const prInfo = prInfoStr(options)
+
+  return (state, travisId, message) => {
+    githubClient.statuses.create({
+      user: options.owner,
+      repo: options.repoName,
+      sha: options.lastSha,
+      target_url: `https://travis-ci.org/${options.owner}/${options.repoName}/builds/${travisId}`,
+      context: "Travis CI via nodejs-github-bot",
+      state: state,
+      description: message,
+    }, (err, res) => {
+      if (err) {
+        return console.error(`! ${prInfo} Error while updating GitHub PR status`, err, err.stack)
```

Comment:
I changed this to also log the whole `err` since the http errors didn't make stacks, but I can remove that.